docs/OV_Converter_UG/prepare_model/convert_model/Convert_Model_From_Paddle.md

```diff
@@ -0,0 +1,201 @@
+# Converting a PaddlePaddle Model {#openvino_docs_OV_Converter_UG_prepare_model_convert_model_Convert_Model_From_Paddle}
+
+@sphinxdirective
+
+.. meta::
+   :description: Learn how to convert a model from the
+                 PaddlePaddle format to the OpenVINO Model.
+
+This page provides general instructions on how to convert a model from a PaddlePaddle format to the OpenVINO IR format using OpenVINO model conversion API. The instructions are different depending on PaddlePaddle model format.
+
+.. note:: PaddlePaddle model serialized in a file can be loaded by ``openvino.Core.read_model`` or ``openvino.Core.compile_model`` methods by OpenVINO runtime API without preparing OpenVINO IR first. Refer to the :doc:`inference example <openvino_docs_OV_UG_Integrate_OV_with_your_application>` for more details. Using ``openvino.convert_model`` is still recommended if model load latency matters for the inference application.
+
+Converting PaddlePaddle Model Files
+###################################
+
+PaddlePaddle inference model includes ``.pdmodel`` (storing model structure) and ``.pdiparams`` (storing model weight). For how to export PaddlePaddle inference model, please refer to the `Exporting PaddlePaddle Inference Model <https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/beginner/model_save_load_cn.html>`__ Chinese guide.
+
+To convert a PaddlePaddle model, use the ``ovc`` or ``openvino.convert_model`` and specify the path to the input ``.pdmodel`` model file:
+
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+      .. code-block:: py
+
+      import openvino as ov
+      ov.convert_model('your_model_file.pdmodel')
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: sh
+
+      ovc your_model_file.pdmodel
```

Comment:
```suggestion
      .. code-block:: py

         import openvino as ov
         ov.convert_model('your_model_file.pdmodel')

   .. tab-item:: CLI
      :sync: cli

      .. code-block:: sh

         ovc your_model_file.pdmodel
```

Reply:
sorry, my bad, added extra spaces and broke the blocks on this page 
http://openvino-doc.iotg.sclab.intel.com/mo_docs/openvino_docs_OV_Converter_UG_prepare_model_convert_model_Convert_Model_From_Paddle.html